seedpod_ground_risk/main.py

```diff
@@ -243,10 +247,13 @@ def layer_reorder(self):
 
     def add_aircraft(self):
         from seedpod_ground_risk.ui_resources.aircraft_options import add_aircraft
-        wizard = AircraftWizard(self, Qt.Window)
-        wizard.exec()
-        ac = wizard.d
-        add_aircraft(ac)
+        try:
+            wizard = AircraftWizard(self, Qt.Window)
+            wizard.exec()
+            ac = wizard.d
+            add_aircraft(ac)
+        except:
```

Comment:
What exception does this throw? Can you handle only that exception on it's own? This kind of construct is just begging for a hidden bug somewhere